crowdstrikeoauthapi.json

```diff
@@ -606,6 +606,11 @@
                         "i-058fca45cda978e00"
                     ]
                 },
+                {
+                    "data_path": "action_result.data.*.kernel_version",
+                    "data_type": "string",
+                    "example_value": "10.0.19044.1766"
```

Comment:
It should be List and key is "example_values" not "example_value".
```
"example_values": [
    "10.0.19044.1766"
]
```
Please check for all the occurrences.